src/sentry/static/sentry/app/views/issueList/savedSearchMenu.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
```

Comment:
The savedSearchMenu is now just the dropdown components shared between the original dropdown button and the savedSearch tab being added in this PR